kv-components/vue/KvSelect.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<div class="inline-flex">
+		<div class="relative w-full">
+			<select
+				:id="id"
+				:disabled="disabled"
+				:value="value"
+				class="text-base h-6 pr-4 pl-1 border border-gray-300 rounded-sm appearance-none w-full"
+				:class="{ 'opacity-low': disabled }"
+				@change="onChange"
+			>
+				<slot></slot>
+			</select>
+			<kv-material-icon
+				:icon="mdiChevronUp"
+				class="absolute right-0 pt-1.5 pr-1 pointer-events-none"
+				:class="{ 'opacity-low': disabled }"
+			/>
+		</div>
+	</div>
+</template>
+
+<script>
+import { mdiChevronUp } from '@mdi/js';
+import KvMaterialIcon from './KvMaterialIcon.vue';
+
+export default {
+	components: {
+		KvMaterialIcon,
+	},
+	// v-model will change when select value updates
+	model: {
```

Comment:
This block is unnecessary

Reply:
@eddieferrer This is needed to get v-model working. Without it, `exampleModel` does not update when the select is changed in the example below.
```
	data: () => ({
		exampleModel: 'example2',
	}),
	template: `
		<div>
			exampleModel = {{exampleModel}}
			<kv-select
				v-model="exampleModel"
			>
					<option value="example1">Example one</option>
					<option value="example2">Example two</option>
					<option value="example3">Example three</option>
			</kv-select>
		</div>`,

Reply:
You're totally right, I thought this was the default, but I think the default is input and not change.